core/src/main/scala/org/bykn/bosatsu/PackageCustoms.scala

```diff
@@ -224,4 +233,65 @@ object PackageCustoms {
       case Some(nel) => Validated.invalid(nel)
     }
   }
-}
+
+  private def noUselessBinds[A: HasRegion](pack: Package.Typed[A]): ValidatedNec[PackageError, Unit] = {
+    type Node = Either[pack.exports.type, Bindable]
+    implicit val ordNode: Ordering[Node] =
+      new Ordering[Node] {
+        def compare(x: Node, y: Node): Int =
+          x match {
+            case Right(bx) =>
+              y match {
+                case Left(_) => 1
+                case Right(by) =>
+                  Ordering[Identifier].compare(bx, by)
+              }
+            case Left(_) => 
+              y match {
+                case Left(_) => 0
+                case Right(_) => -1
+              }
+          }
+      }
+
+    val exports: Node = Left(pack.exports)
+    val roots: List[Node] =
+      (exports ::
```

Comment:
Not: put exports last.

Reply:
This isn't needed. I'm being ticky here and `exports` is a single node, not a list.